index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width , initial-scale=1.0">
+		<title>Personal Portfolio</title>
+		
+		<link rel="stylesheet" type="text/css" href="styles.css" />
+		<link rel="preconnect" href="https://fonts.googleapis.com"/>
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet"/>
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet" />
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet" />
+	</head>
+	<body>
+	   
+			 <header class="header">
+			   <nav> 
+					<ul class="container menu">
+				<li><a href="#"><h3 class="name">John Doe</h3></a></li> 
```

Comment:
- [ ] [OPTIONAL] Good job implementing the requested changes. It would be great if you use your own name or initials instead of `John Doe` so users can know whose portfolio this is. 😎️

Reply:
hey amejid,
Thank you for reviewing my code, so I am going to replace the name john Doe with my name. Thanks for the positive feedback